pandas/tests/indexes/test_interval.py

```diff
@@ -491,6 +649,207 @@ def testcontains(self):
         assert not i.contains(20)
         assert not i.contains(-20)
 
+        # Why are there two tests called test contains? this should be cleared up a little, no?
+
+
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_interval_covers_interval_updated_behavior(self):
+
+        # class Interval:
+        #     def covers(self, other: Interval) -> bool
```

Comment:
Just that I understand correctly: this is a proposal for the behaviour of a new method?